bin/s2nd.c

```diff
@@ -41,6 +41,8 @@
 #include <s2n.h>
 #include "common.h"
 
+#include "tls/s2n_connection.h"
+#include "tls/s2n_tls13.h"
```

Comment:
s2n_tls13.h no longer needs to be included.

Reply:
Neither did `s2n_connection.h`, removed both.